.travis.yml

```diff
@@ -102,10 +113,16 @@ script:
   - make && sudo env PATH=$PATH make install && awesome --version
   - |
     if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
-      if [ "$DO_COVERAGE" = "true" ]; then
+      do_codecov samples
+    fi
+  - |
+    if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
+      if [ "$DO_COVERAGE" != "0" ]; then
```

Comment:
You could test for `-n "$DO_COVERAGE` here, and then just not export it as `0` by default above.